### PR TITLE
SDK-188: Created interfaces for violation

### PR DESCRIPTION
**Pull request by vol4onok**

### CODE REVIEW

File: src/Entity/CommandInterface.php [hunk 1]
```diff
@@ -28,4 +28,9 @@ public function getTags(): array;
      * @return bool
      */
     public function hasStopOnError(): bool;
+
+    /**
+     * @return \SprykerSdk\SdkContracts\Entity\ConverterInterface|null
+     */
+    public function getConverter(): ?ConverterInterface;
```
Comment (dereuromark) [inline src/Entity/CommandInterface.php]: Do they have to be nullable?
This makes using them impossible directly, they always need to check for !== null before using.

An alternative approach could be a has*() method so the getter stays clean and directly usable, in line with PHPStan level 8 etc.

Comment (vol4onok) [reply src/Entity/CommandInterface.php]: not all commands have convertors

Comment (dereuromark) [reply src/Entity/CommandInterface.php]: Sure, but then consider my recommendations for clean API usage

Comment (vol4onok) [reply src/Entity/CommandInterface.php]: if I add has*() how I implement getConverter(): ConverterInterface; if I don't have this convertor in entity. has*() doesn't make sense here

Comment (dereuromark) [reply src/Entity/CommandInterface.php]: Yeah, get*orFail() is the correct approach for DTOs
but it can also be added later
so no blocker

**Review by newLoki — APPROVED** [review]

**Pull request merged**